src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec

```diff
@@ -0,0 +1,86 @@
+# do not repack jars
+%define __jar_repack %{nil}
+# produce .elX dist tag on both centos and redhat
+%define dist %(/usr/lib/rpm/redhat/dist.sh)
+
+Name:               xroad-proxy-ui-api
+Version:            %{xroad_version}
+# release tag, e.g. 0.201508070816.el7 for snapshots and 1.el7 (for final releases)
+Release:            %{rel}%{?snapshot}%{?dist}
+Summary:            X-Road proxy UI REST API
+Group:              Applications/Internet
+License:            MIT
+BuildRequires:      systemd
+Requires(post):     systemd
+Requires(post):     /usr/sbin/semanage, /usr/sbin/setsebool
+Requires(preun):    systemd
+Requires(postun):   systemd
+Requires:           net-tools, tar
```

Comment:
xroad-proxy-ui-api-setup.sh needs `ip` and `hostname` (from _iproute_ and _hostname_ packages). net-tools should be unnecessary. Can't find any reference to `tar` either.

Reply:
fixed